unit-tests-jest/numbers.ts

```diff
@@ -33,8 +38,8 @@ export function multiplyBy(
 ): Record<string, unknown> {
   const result: Record<string, unknown> = {};
   Object.entries(obj).forEach(([key, value]) => {
-    if (Number.isInteger(value)) {
-      result[key] = Number(value) * multiplier;
+    if (!Number.isNaN(Number(value))) {
+      result[key] = Math.round(Number(value) * multiplier * 100) / 100;
```

Comment:
I don't understand this math? Why not just `Number(value) * multiplier`? (Or, since you know it's a number, just `value * multiplier`.)